src/item_factory.cpp

```diff
@@ -3782,6 +3789,11 @@ void Item_factory::load_basic_info( const JsonObject &jo, itype &def, const std:
     assign( jo, "solar_efficiency", def.solar_efficiency );
     assign( jo, "ascii_picture", def.picture_id );
 
+    if( jo.has_member( "repairs_with" ) ) {
+        def.repairs_with.clear();
+        optional( jo, false, "repairs_with", def.repairs_with );
+    }
```

Comment:
```suggestion
    optional( jo, false, "repairs_with", def.repairs_with, string_id_reader<material_type>() );
```
Sorry for the misleading tip, this is all you need for copy-from